utils/torch_solver.py

```diff
@@ -34,7 +39,7 @@ def run(self, callback):
         # Setup the trainer
         trainer = Trainer(
             max_epochs=-1, callbacks=[BenchoptCallback(callback)],
-            accelerator="gpu" if torch.cuda.is_available() else None
+            accelerator="auto", devices=1
```

Comment:
here we'll probably need to have a nice way to check the number of devices

Reply:
We can see this later on.